src/process/plumerise/CCPr_Scheme_Sofiev_mod.F90

```diff
@@ -3,6 +3,7 @@
 !!!>
 module ccpr_scheme_sofiev_mod
 
+   use precision_mod, only: rae
```

Comment:
do we want to put this in precision_mod or should we have this in a utility or something.  Just thinking that we want to keep this out of the core perhaps for these schemes themselves

Reply:
I thought it made sense there since it is precision-related. I do feel it should live somewhere in core, it's applicable to more than just the schemes.

Reply:
Well the problem is we have basically stripped anything from the core from the schemes themselves.  I guess the question is should this live in something separate from the core (this goes with most things in the precision_mod not just this single variable).